client/packages/system/src/Manage/Preferences/EditPage/EditPreferencesPage.tsx

```diff
@@ -1,36 +1,37 @@
-import React, { useEffect } from 'react';
+import React from 'react';
 import {
+  Box,
+  InputWithLabelRow,
   LocaleKey,
-  useBreadcrumbs,
-  useParams,
+  NothingHere,
+  PreferenceNodeType,
   useTranslation,
 } from '@openmsupply-client/common';
-import { LineEditBase } from './LineEditBase';
-import { useAvailablePreferences } from '../api';
 import { EditPreference } from './EditPreference';
+import { useEditPreferences } from '../api/useEditPreference';
 
 export const EditPreferencesPage = () => {
   const t = useTranslation();
-  const { key } = useParams();
-  const { setCustomBreadcrumbs } = useBreadcrumbs();
 
-  const { data } = useAvailablePreferences();
+  const { update, preferences } = useEditPreferences(PreferenceNodeType.Global);
 
-  useEffect(() => {
-    if (key)
-      setCustomBreadcrumbs({
-        1: t(`preference.${key}` as LocaleKey),
-      });
-  }, [key]);
-
-  const selectedPref = data?.find(d => key === d.key);
+  if (!preferences.length) return <NothingHere />;
 
   return (
-    // TODO: Consider alternative UI - more explicit than JSON forms?
-    <LineEditBase currentKey={key ?? ''} prefs={data ?? []}>
-      {selectedPref && (
-        <EditPreference key={selectedPref.key} selected={selectedPref} />
-      )}
-    </LineEditBase>
+    <Box display="flex" justifyContent="center" width="100%" marginTop={2}>
+      <Box width="600px">
+        {preferences.map(pref => (
+          <InputWithLabelRow
+            key={pref.key}
+            label={t(`preference.${pref.key}` as LocaleKey)}
```

Comment:
I wonder if you could use `translateDynamicKey` from IntlUtils.ts with manual fallback string?

This way missing translation key will be a bit more nice than 'preferences.key' showing in UI

Reply:
Hm... I like the idea, but I don't really think prefs without a related translation is a valid case we'd want to support. If you're adding a new pref, it should have a matching translation key. Kind of think the ugly preference.key is good to keep around, reminds you there is something left to do!

Reply:
> Hm... I like the idea, but I don't really think prefs without a related translation is a valid case we'd want to support. If you're adding a new pref, it should have a matching translation key. Kind of think the ugly preference.key is good to keep around, reminds you there is something left to do!

Yeah nice good point!